README.md

```diff
@@ -85,7 +85,8 @@ To run what's in this repository, you need to first start a Codespaces instance.
 1. Navigate to the main page of the newly created repository.
 1. Under the repository name, use the Code drop-down menu, and in the Codespaces tab, select "Create codespace on main".
 
-    <img src="https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png" alt="Create Codespace" style="width: 270px;"/>
+    <img src="https://github.com/user-attachments/assets/13f0af8d-7fe0-4a1e-a559-244db79c78cd" alt="Create Codespace" style="width: 600px;" />
```

Comment:
where is this user attachment coming from?

Reply:
Hi @alfredodeza 
The file is uploaded to GitHub's storage and linked to the PR for reference,

Reply:
Here is the attachment that was uploaded to GitHub storage
![create codespace](https://github.com/user-attachments/assets/0b29d55a-a55e-4cc8-b44f-c477b84ad713)

Reply:
Right, since this is going to a user storage, then the account can be removed/changed or the file simply deleted. Although your change would help address this, it is not a long-term solution which would be preferable